conf/evolutions/default/221.sql

```diff
@@ -0,0 +1,13 @@
+# --- !Ups
+CREATE TABLE panorama_history (
+   panorama_id TEXT NOT NULL,
+   visited_timestamp TIMESTAMP,
+   pano_month INT NOT NULL,
+   pano_year INT NOT NULL,
```

Comment:
Is there a reason for storing month and year separately rather than storing the date in the same way that we store dates in the `gsv_data` table? We should probably be consistent unless there's a specific reason.

Reply:
I was just thinking that if we wanted to write a query to get panos for a specific month or specific year that format might be easier since this table is more for us internally and the API doesn't really give us a day of the month or anything. That being said if it won't make a difference in query convenience I'm all for changing it back!

Reply:
Yeah I don't see it being something that we want to do frequently, and it would be easy enough to get the month/year out of the string in SQL too